scripts/pysa_fuzzer/code_generator2.py

```diff
@@ -0,0 +1,194 @@
+from typing import List
+import itertools
+import string
+
+class CodeGenerator:
+    def __init__(self) -> None:
+        self.variables = self.generate_variable_names()
+        self.current_var = 1
+        self.current_function_number = 1
+        self.source_statements = ['a = input()']
+        self.last_source = "a"
+        self.sink_statements = []
+        self.last_sink = "print"
```

Comment:
It would be more interesting to start with:
```suggestion
        self.source_statements = []
        self.last_source = "input()"
        self.sink_statements = []
        self.last_sink = "print"
```
In a way, storing the taint in a temporary variable is already a mutation.